v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/utils/SchemaUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.spanner.migrations.utils;
+
+import com.google.cloud.teleport.v2.spanner.migrations.exceptions.DroppedTableException;
+import com.google.cloud.teleport.v2.spanner.migrations.schema.Schema;
+
+/**
+ * Class containing util methods that use the {@link
+ * com.google.cloud.teleport.v2.spanner.migrations.schema.Schema} object.
+ */
+public class SchemaUtils {
```

Comment:
Is it logically possible to rename this into something more specific? I would like to avoid `util` classes as much as possible (refer to my previous comment)

Reply:
How about SchemaHelper then?

Reply:
The latter seems better to me too!